### PR TITLE
Fix bug 1427048: Fix tab shortcut regression

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes r?

### CODE REVIEW

File: pontoon/base/static/js/translate.js [hunk 1]
```diff
@@ -1853,13 +1853,13 @@ var Pontoon = (function (my) {
...
-          // Prevent in complex FTL mode
+          // Rich FTL editor: ignore tab key
           if (self.fluent.isFTLEditorEnabled() && self.fluent.isComplexFTL()) {
             return;
           }
 
-          // Special case in FTL source editor
-          if ($('#translation').is(':visible') || $('#ftl').is('.active')) {
+          // Source FTL editor: insert tab character
+          if ($('#ftl:visible').is('.active')) {
```
Comment (jotes) [inline pontoon/base/static/js/translate.js]: isn't that `self.fluent.isComplexFTL()`?

Comment (mathjazz) [reply pontoon/base/static/js/translate.js]: No, FTLEditor is the form-based AKA rich FTL editor as seen here:
https://pontoon.mozilla.org/sl/firefox/browser/browser/preferences/main.ftl/?string=174655

**Comment by jotes** [comment]
Ok, got it.  [Nit] Can we return that 'state' of app by some method?

**Comment by mathjazz** [comment]
Updated with a method.

**Pull request merged**